### ISSUE TITLE
Option to set "Series Type".

### ISSUE THREAD

**Issue by yuvsama001** [komikku-app/komikku#217]
### Describe your suggested feature

If this feature is implanted then it will manually solve the problem of "Reading Direction/Way", In some sources the Tags aren't shows so the tags won't display. So, it sometimes don't Auto adapted the webtoon as default for that specific Manhwa/hua.

Then this feature come in as to set default "Reading Way" for specifics Entry.

For example:- 
1. If series type is "Comic" then the reading way will be set default to "Left to Right".

2. If series type is "Manga" then the reading way will be set default to "Right to Left".

3. If series type is "Manhwa" then the reading way will be set default to "Long Strip".

4. If series type is "Manhua" then the reading way will be set default to "Long Strip".

5. If series type is "Webtoon" then the reading way will be set default to "Long Strip".

### Other details

![Screenshot_2024-08-02-19-30-03-61_21015a41200050f19d2d2db913bd7dc1](https://github.com/user-attachments/assets/f29c3704-996e-41ed-927a-2176bbe9e549)
![Screenshot_2024-08-02-19-58-02-21_21015a41200050f19d2d2db913bd7dc1](https://github.com/user-attachments/assets/df136216-529e-477c-ab98-9dfe8d769a0d)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an official extension, I should be opening an issue in the [extensions repository](https://github.com/komikku-app/extensions/issues/new/choose).
- [X] I have updated the app to version **[1.11.1](https://github.com/komikku-app/komikku/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by JoeloJoestar** [comment]
You can already set up the reading direction per series through the reading mode settings in reading screen, so this proposed manual option in the edit window is a futile redundancy which won't improve the Quality of Life.
What might help is to have the ability to set a default reading direction per source and then manually adjust the exceptional entries, because whatever the tag many sources do publish stuff from both west and east.. Some entry tagged as Manga when their not, and even the actual manga might be oriented left-to-right (like old Officially English published).

**Comment by yuvsama001** [comment]
> You can already set up the reading direction per series through the reading mode settings in reading screen, so this proposed manual option in the edit window is a futile redundancy which won't improve the Quality of Life. What might help is to have the ability to set a default reading direction per source and then manually adjust the exceptional entries, because whatever the tag many sources do publish stuff from both west and east.. Some entry tagged as Manga when their not, and even the actual manga might be oriented left-to-right (like old Officially English published).

I already know that, but what I want is that it set it to default for that specific entry.

**Comment by cuong-tran** [comment]
is the screenshort from Yokai?
is there any other usage for the "series type" beside setting reading mode? because manually setting series type would be same as manually set reading mode if there is nothing else.

**Comment by yuvsama001** [comment]
> is the screenshort from Yokai? is there any other usage for the "series type" beside setting reading mode? because manually setting series type would be same as manually set reading mode if there is nothing else.

Yeah there is, it will allowed users to filter out Comic, Manga, and Webtoon as required.

**Comment by isabella-almeidaa** [comment]
please if you plan on adding this, make it so the reading way by tag is configurable... Personally, I just use Long Strip (for manhwa/manhua/webtoon) and long strip with gaps (for everyting else)
